docs/versioning.md

```diff
@@ -81,7 +81,7 @@ Antrea maintains release branches for the two most recent minor releases
 0.12 is released). As part of this maintenance process, patch versions are
 released as frequently as needed, following these
 [guidelines](#minor-releases-and-patch-releases). With the current release
-cadence, this means that each minor release receives approximately 3 months of
+cadence, this means that each minor release receives approximately 6 months of
```

Comment:
@antoninbas if we want to extend this to 9 months, I suppose we will maintain three most recent minor releases?

Reply:
yes, but we don't have to make that decision now

I wish we had a more efficient way of doing patch releases, it feels like we are not doing them quite often enough.